data/json/npcs/talk_tags.json

```
@@ -259,7 +313,12 @@
       "I'm done for",
       "I won't last much longer",
       "my days are <really> numbered",
-      "<name_g>, I'm sorry"
+      "<name_g>, I'm sorry",
+      "Didn't think it would end like this.",
+      "So, this is how it ends, huh?",
+      "...I'll be with you soon Mary...I'm coming...",
```

```suggestion
      "...I'll be with you soon, Mary... I'm coming...",
```
I don't like this one. It relies on punctuation where punctuation isn't allowed, and I don't like how any NPC can potentially know someone called "Mary."

---

It was an attempt to give the NPC a semblance of depth, like they actually knew people outside of our reality bubble, had people they lost, but whatever, it's gone.